### PR TITLE
Add IBM command set

### PR DESCRIPTION
**Pull request by satyarohith**
Includes commands to interact with IBM Cloud Functions & IBM Watson services.

### CODE REVIEW

File: ibm/packages/functions/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# IBM Cloud Functions
```
Comment (niks3089) [inline ibm/packages/functions/README.md:1]: Do we need multiple README files? Can we put this in the top README?

File: ibm/packages/functions/invoke.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+ */
+function getParams(string = '') {
+  const object = {};
+
+  for (const str of string.split('-')) {
+    if (str === '') {
+      continue;
+    }
+
+    // `name "Satya Rohith"` => ["name", "Satya Rohith"]
```
Comment (niks3089) [inline ibm/packages/functions/invoke.js]: Please remove a generic name like "bob" etc

File: ibm/packages/tone_analyzer/tone.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      text: 'Overall'
+    }
+  });
+
+  const documentTone = {
+    type: 'context',
+    elements: []
+  };
+
+  for (const tone of data.document_tone.tones) {
```
Comment (niks3089) [inline ibm/packages/tone_analyzer/tone.js:51]: What the maximum number of tones that could be returned?

File: ibm/packages/translate/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  "url": "..."
+}
+```
+
+Head over to the Secret Creator (type `/nc secret_create` in your workspace to get the link) and fill in the *Name* field with `ibmLanguageTranslatorCredentials` and paste the copied values in the corresponding *Value* field. Press **Make Secrets** button and follow the instructions.
+
+## Usage
+
+```sh
+/nc translate <text> <targetLanguage>
```
Comment (niks3089) [inline ibm/packages/translate/README.md]: We already have a translate command. I guess the error will pop up saying command already exists?

File: ibm/packages/functions/README.md (hunk 1 above)
Comment (satyarohith) [reply ibm/packages/functions/README.md:1]: Yes, the requirements are different for translate and tone.

File: ibm/packages/tone_analyzer/tone.js (hunk 3 above)
Comment (satyarohith) [reply ibm/packages/tone_analyzer/tone.js:51]: It depends on the text, so looping.

File: ibm/packages/translate/README.md (hunk 4 above)
Comment (satyarohith) [reply ibm/packages/translate/README.md]: Renamed it to `traducir`.

File: ibm/packages/functions/invoke.js (hunk 2 above)
Comment (satyarohith) [reply ibm/packages/functions/invoke.js]: Sorry, I didn't get you. Should I use a different example?

Comment (niks3089) [reply ibm/packages/functions/invoke.js]: Don't use your name. Instead, use "bob" or something generic.

Comment (satyarohith) [reply ibm/packages/functions/invoke.js]: Updated!

**Review by niks3089 — APPROVED** [review]

**Pull request merged**